### ISSUE TITLE
[Python] Implement new Concept Exercise: in

### ISSUE THREAD

**Issue by DavidGerva** [exercism/v3#1431]
This issue describes how to implement the `membership testing 'in'` operator concept exercise for the Python track.

## Getting started

**Please please please read the docs before starting.** Posting PRs without reading these docs will be a lot more frustrating for you during the review cycle, and exhaust Exercism's maintainers' time. So, before diving into the implementation, please read up on the following documents:

- [The features of v3](https://github.com/exercism/v3/blob/master/docs/concept-exercises.md).
- [Rationale for v3](https://github.com/exercism/v3/blob/master/docs/rationale-for-v3.md).
- [What are concept exercise and how they are structured?](https://github.com/exercism/v3/blob/master/docs/features-of-v3.md)

Please also watch the following video:

- [The Anatomy of a Concept Exercise](https://www.youtube.com/watch?v=gkbBqd7hPrA).

## Goal

This concept exercise should convey a basic understanding of the `in` operator in Python to perform a membership testing.

## Learning objectives

- use the `in` operator to tests if an object is (not) a member of a collection

## Out of scope

- identity testing via `is` and `is not`
- comparisons

## Concepts

- `in` operator
- membership

## Prerequisites

- `basics`
-  booleans, `True`, `False`
- `dicts`
- `lists`
- `sets`
- `tuples`

## Resources to refer to

- [The in operator](https://yawpitchroll.com/posts/the-35-words-you-need-to-python/#in)
- [Membership testing (Python docs)](https://docs.python.org/3/reference/expressions.html#membership-test-operations)

### Hints

- One or more of the links above or analogous links from a trusted source.

### After

Nothing

## Representer

No changes required.

## Analyzer

No changes required.

## Implementing

Tests should be written using unittest.TestCase and the test file named in_test.py.

## Help

If you have any questions while implementing the exercise, please post the questions as comments in this issue.

## Edits
-

**Comment by aimorris** [comment]
This should be in the `collections` concept exercise where you can explain everything about collections, including the `in` keyword.

**Comment by BethanyG** [comment]
Ooof.  @DavidGerva -- After going back over this twice, I have to agree with @aimorris - we should do membership checking alongside each data structure, and not try to making it it's own exercise.  Having checked `tuples` and `lists` -- we're already doing that, so I am going to close this as `no longer relevant`.  Apologies for having you do extra work.  😦 .

**Comment by BethanyG** [comment]
If there is a strong disagreement, please re-open.

**Comment by SleeplessByte** [comment]
(I agree with the decision of adding `in` to each data structure's exercise.)
